pynuodb/encodedsession.py

```diff
@@ -257,6 +282,11 @@ def send_close(self):
         self._exchangeMessages()
         self.close()
 
+    def __set_dbinfo(self, sid, txid, seqid):
+        # type: (int, int, int) -> None
+        with EncodedSession.__dblock:
+            EncodedSession.__databases[str(self.db_uuid)].info[sid] = (txid, seqid)
```

Comment:
This is wrong.  We only want to update the values if the new sequence ID is greater than the old one (at least, that's how it works in the C/C++ drivers.